libs/ui/src/lib/language-indicator/language-indicator.module.css

```diff
@@ -0,0 +1,47 @@
+.container {
+  display: inline-block;
+  min-width: 1em;
+  border-radius: 2px;
+  padding: 0 0.25em;
+}
+
+.label {
+  text-align: center;
+  font-weight: 700;
+  font-size: 1rem;
+}
+
+.containerDataset {
+  composes: container;
+  background-color: rgb(213, 225, 242);
+}
+
+.containerDataservice {
+  composes: container;
+  background-color: rgb(242, 225, 213);
+}
+
+.containerConcept {
+  composes: container;
+  background-color: rgb(213, 237, 242);
+}
+
+.containerInformationmodel {
```

Comment:
Tror at vi går vekk fra å bruke forskjellige farger for de ulike typene i nye designet. Hør med Matheus men tror vi skal bort fra det.